[1.1.1] mapArray 배열 생성

mellime opened the pull request:
1. ui.h에서  맵을 그릴 때 사용할, 맵의 형태를 잡은 2차원 문자 배열 mapArray를 선언. extern으로 선언 뒤, ui.c에서 구현.
2. 최근 머지 내역 패치 후, 충돌 해결
3. 맵을 그릴 때 사용하는 문자를 변경하여 인코딩에 따라 오류가 발생하는 상황을 해결

mellime committed:
[NEW] Creating map array

I declared two dimensional array of characters to use when drawing map in ui.h file.
I implemented the array in ui.c file.

SonHyeWon commented:
원래 빌드 오류 나는건가요? 우선 추가하신 코드는 확인 했습니다.

MtNight commented:
엉... 일단 확인했습니다. 저는 sprint문제 외에는 빌드 오류가 안나긴 하는데, 그냥 변수 하나 추가한거라 오류가 안나야 하지 않나요?

mellime commented:
@SonHyeWon 추가한 코드에 대해서는 오류가 나지 않는게 맞는데, 혹시 어떤 오류가 발생하는지 알 수 있을까요?

mellime committed:
[MOD] Resolving conflict error

I resolved conflict error in ui.c file.

hongyj9823 commented:
확인 완료했습니다. 컴파일 잘됩니다.

SonHyeWon commented:
> @SonHyeWon 추가한 코드에 대해서는 오류가 나지 않는게 맞는데, 혹시 어떤 오류가 발생하는지 알 수 있을까요?

Issue #27 에서 의견 합의가 이루어진 듯 합니다. 사용 문자를 바꾼 후 다시 풀리퀘스트 부탁드려도 될까요?

mellime commented:
> Issue #27 에서 의견 합의가 이루어진 듯 합니다. 사용 문자를 바꾼 후 다시 풀리퀘스트 부탁드려도 될까요?

네. 수정 후 다시 푸시하도록 하겠습니다.

mellime committed:
Merge remote-tracking branch 'origin/master' into new_createMapArray

mellime committed:
[MOD] Changing type of characters

I changed type of characters used to draw map in ui.c file.
I changed size of array from 240 to 120 in ui.c and ui.h file.

mellime commented:
@SonHyeWon 사용 문자를 변경하였습니다. 다시 한 번 확인 부탁드립니다.

MtNight commented:
확인했습니다

SonHyeWon commented:
확인했습니다.
